src/js/exporters.js

```diff
@@ -272,6 +276,57 @@ function exportKontaktScript() {
   return true
 }
 
+function exportImageLinePitchMap(range) {
+  if (exportError()) {
+    return
+  }
+  const NB_NOTES = 121 // IL products can only retune from C0 to C10
+  const HEADER_BYTES = Uint8Array.from([3, 0, 0, 0, 3, 0, 0, 0, NB_NOTES, 0, 0, 0])
+  const ENDING_BYTES = Uint8Array.from([0, 0, 0, 0, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255])
+
+  const tuningTable = model.get('tuning table')
+  const baseFreqOffset = Math.log2(tuningTable.baseFrequency / 440) // in number of octaves
+
+  // construct point data
+  let points = new ArrayBuffer(121 * 24)
+  let pointsDoubles = new Float64Array(points)
+  let pointsUint32 = new Uint32Array(points)
+  for (let i = 0; i < NB_NOTES; i++) {
+    const edo12cents = (i - 69) * 100
+    const offset = tuningTable.cents[i] - edo12cents
+    const normalizedOffset = ((offset / 1200 + baseFreqOffset) / range) * 0.5 + 0.5
+    const yCoord = Math.max(0.0, Math.min(1.0, normalizedOffset))
```

Comment:
My eyes cought this line of code, which could be a simplified a bit using the [clamp](https://github.com/SeanArchibald/scale-workshop/blob/develop/src/js/helpers/numbers.js#L233) function from `helpers/numbers.js`:

```javascript
const yCoord = clamp(0, 1, normalizedOffset)
```